operations/iscsi_sbps/Node_Personalization_Verification.md

```diff
@@ -0,0 +1,255 @@
+# Node Personalization Verification
+
+* [Introduction](#introduction)
+* [Post-personalization Verification](#post-personalization-verification)
+    * [Check SBPS Marshal Agent status](#check-sbps-marshal-agent-status)
+    * [Check `targetcli ls` command output](#check-targetcli-ls-output)
+    * [Check for DNS SRV and A records](#check-for-dns-srv-and-a-records)
+    * [Run readiness checks](#run-readiness-checks)
+
+## Introduction
+
+Worker node personalization installs SBPS Marshal agent, adds `kubernetes` labels for the worker NCNs intended to be configured as iSCSI targets, it also removes the labels from workers that are not designated to be iSCSI targets, creates DNS
+SRV and A records etc. See [Managing selective node personalization](Managing_Selective_Node_Personalization.md) for details on how worker NCNs designated to be enabled as iSCSI targets.
+
+## Post-personalization Verification
+
+(`ncn-w#`) Run these checks on worker nodes that are enabled as an iSCSI target in order to verify that it is
+ configured correctly.
+
+### Check SBPS Marshal Agent status
+
```

Comment:
Some introductory information is needed here about the SBPS Marshal Agent. What it is and what it does. If this information is already described on the main iSCSI page, then here you can just give a short summary (of the information needed to understand the check procedure) and then link back to the full info.